### ISSUE TITLE
[AIR 33.1.0.37 beta][Windows][macOS] Crash with Embed images, audio or video

### ISSUE THREAD

**Issue by itlancer** [Gamua/Adobe-Runtime-Support#227]
## Problem Description
Application crashes with AIR 33.1.0.37 beta on Windows and macOS devices if `Embed` used for images, audio or video files.

It has been tested with latest AIR 33.1.0.37 beta with Windows and macOS devices. Tested with different images, audio and video files.
Same problem in all cases. But there is no such crash if you try to embed fonts.

It works fine with Android.
Also It works fine with AIR 33.1.0.16 beta and below with any platform.
Doesn't check AIR 33.1.0.37 beta yet with iOS and tvOS.

Related (not the same) issues:
https://github.com/Gamua/Adobe-Runtime-Support/issues/162
https://github.com/Gamua/Adobe-Runtime-Support/issues/153
https://github.com/Gamua/Adobe-Runtime-Support/issues/150

## Steps to Reproduce
Launch code below with Windows or macOS device with AIR 33.1.0.37 beta.

Application example with sources and example of image attached.
[embed_crash.zip](https://github.com/Gamua/Adobe-Runtime-Support/files/4188971/embed_crash.zip)

```
package {
	import flash.display.Sprite;
	
	public class EmbedCrash extends Sprite {
		
		[Embed(source = "image.jpg")]//This line cause crash
		private const EmbeddedImage:Class;
		
		public function EmbedCrash() {
			
		}
	}
}
```

**Actual Result:**
Application crash at start.
Windows crash log:
```
Problem signature:
  Problem Event Name:	APPCRASH
  Faulting application name:	adl.exe
  version:	33.1.0.37
  time stamp:	5e3d9e6a
  Faulting module name:	Adobe AIR.dll
  version:	33.1.0.37
  time stamp:	5e3db542
  Exception code:	c0000005
  Fault offset:	002548c8
  OS Version:	6.3.9600.2.0.0.256.48
  Locale ID:	1049
  Additional Information 1:	5861
  Additional Information 2:	5861822e1919d7c014bbb064c64908b2
  Additional Information 3:	d1d9
  Additional Information 4:	d1d94a13d3609d6b740644c12508f581
```

**Expected Result:**
Application will be launched without crash.

## Known Workarounds
none

**Comment by tuarua** [comment]
This is also happening if I reference a font compiled as an external swc.
eg
`public static const FONT:Font = new FiraSansSemiBold();`

FiraSansSemiBold.swc included as a library

> [AIR Debug Launcher]: Process finished with exit code -1,073,741,819

**Comment by itlancer** [comment]
Fixed with AIR 33.1.0.43 beta. With my assets it doesn't crashes anymore.
Thank you!
